Automattic/wp-calypso
# Full Site Editing: Blank Theme, child of Twenty Nineteen

#### Changes proposed in this Pull Request

* Create a new Blank Theme, child of Twenty Nineteen, that handles how pages with Full Site Editing templates are rendered.

Notes on using Twenty Nineteen styles:
- The default alignment is awkwardly off-centered to the left which causes the template to look weird.
A quick workaround would be assigning a `wide` (which is centered) or `full` alignment to the template part and content slot blocks.
- The main headings have a short line above as a `:before` pseudo element, which is not applied to the site title: `h1:not(.site-title):before, h2:before { ... }`.
I haven't overridden it in the child style because I would argue we should take care of it when we add the Site Title block (and maybe Site Description too).

#### Testing instructions

* Follow the [FSE readme](https://github.com/Automattic/wp-calypso/blob/e759e056360b72576e124989c4174afef78bc1db/apps/full-site-editing/README.md#local-development), and setup both the plugin and the theme.
* Enable the Blank Theme.
* Create some template parts, and assemble them into a template which also contains a Post Content slot.
* Assign this template to a **page**. Save it and view (or preview) it.
* Make sure it looks like it looks in the editor.
* Make sure it looks like a Twenty Nineteen site.
* Make sure nothing blew up in the process.

Fix #33160
